pkg/components/aws-ebs-csi-driver/component.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*Package awsebscsidriver - The golang plumbing for the aws-ebs-csi-driver
+component */
+package awsebscsidriver
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/hcl/v2"
+	"github.com/hashicorp/hcl/v2/gohcl"
+	"github.com/kinvolk/lokomotive/pkg/components"
+	"github.com/kinvolk/lokomotive/pkg/components/util"
+	"github.com/pkg/errors"
+)
+
+const name = "aws-ebs-csi-driver"
+
+const chartValuesTmpl = ``
+
+//nolint:gochecknoinits
+func init() {
+	components.Register(name, newComponent())
+}
+
+type component struct{}
+
+func newComponent() *component {
+	return &component{}
+}
+
+// LoadConfig loads the component config.
+func (c *component) LoadConfig(configBody *hcl.Body, evalContext *hcl.EvalContext) hcl.Diagnostics {
+	if configBody == nil {
+		return hcl.Diagnostics{}
+	}
+
+	return gohcl.DecodeBody(*configBody, evalContext, c)
+}
+
+// RenderManifests renders the helm chart templates with values provided.
+func (c *component) RenderManifests() (map[string]string, error) {
+	helmChart, err := util.LoadChartFromAssets(fmt.Sprintf("/components/%s/manifests", name))
+	if err != nil {
+		return nil, errors.Wrap(err, "load chart from assets")
+	}
+
+	values, err := util.RenderTemplate(chartValuesTmpl, c)
+	if err != nil {
+		return nil, errors.Wrap(err, "render chart values template")
+	}
+
+	renderedFiles, err := util.RenderChart(helmChart, name, c.Metadata().Namespace, values)
+	if err != nil {
+		return nil, errors.Wrap(err, "render chart")
```

Comment:
Please use `fmt.Errorf` instead of `errors.Wrap` for new code. 
```suggestion
		return nil, fmt.Errorf("rendering chart failed: %w", err)
```

See https://github.com/kinvolk/lokomotive/issues/59 for more details.